demos/dqx_demo_library.py

```diff
@@ -153,14 +153,24 @@
       allowed:
         - 1
         - 2
+- criticality: error
+  check:
+    function: is_not_null
+    arguments:
+      col_name: element_at(col5, 'key1')
```

Comment:
What happens if the element is not there. Shouldn't we use `try_element_at` everywhere?